.github/workflows/testing.yml

```diff
@@ -198,7 +198,14 @@ jobs:
         if: matrix.tarantool != 'brew' && matrix.tarantool != 'master'
 
       - name: Install tarantool build dependencies
-        run: brew install autoconf automake libtool
+        run: |
+          brew install autoconf automake libtool openssl@1.1
+          brew link --force openssl@1.1
```

Comment:
Whether we need this `brew link` command or just cmake options below are sufficient?

Reply:
CMake options seem to be sufficient here. Fixed.